generative_ai/extensions/create_example.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from vertexai.preview import extensions
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def create_extension() -> extensions.Extension:
+    # [START generativeaionvertexai_create_extension]
+    import vertexai
+    from vertexai.preview import extensions
+
+    # TODO(developer): Update and un-comment below line
+    # PROJECT_ID = "your-project-id"
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    extension = extensions.Extension.create(
+        display_name="Code Interpreter",
+        description="This extension generates and executes code in the specified language",
+        manifest={
+            "name": "code_interpreter_tool",
+            "description": "Google Code Interpreter Extension",
+            "api_spec": {
+                "open_api_gcs_uri": "gs://vertex-extension-public/code_interpreter.yaml"
+            },
+            "auth_config": {
+                "google_service_account_config": {},
+                "auth_type": "GOOGLE_SERVICE_ACCOUNT_AUTH",
+            },
+        },
+    )
+    print(extension.resource_name)
+    # Example output:
```

Comment:
Minor Nit:  can you use & updates other too please?
```suggestion
    # Example response:
```

In the future, when we conduct a `grep` search for "`Example response`", we can promptly locate the samples that lack the same response.

Reply:
Agree with you! Fixed it👍